src/main/kotlin/app/revanced/patches/spotify/lite/ondemand/fingerprints/OnDemandFingerprint.kt

```diff
@@ -0,0 +1,23 @@
+package app.revanced.patches.spotify.lite.ondemand.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.Opcode
+
+object OnDemandFingerprint : MethodFingerprint(
+    "L",
+    parameters = listOf(),
+    opcodes = listOf(
+        Opcode.INVOKE_STATIC,
+        Opcode.MOVE_RESULT,
+        Opcode.INVOKE_STATIC,
+        Opcode.MOVE_RESULT_OBJECT,
+        Opcode.IF_EQZ,
+        Opcode.SGET_OBJECT,
+        Opcode.GOTO,
+        Opcode.SGET_OBJECT,
+        Opcode.INVOKE_VIRTUAL,
+        Opcode.MOVE_RESULT,
+        Opcode.IPUT,
+        Opcode.RETURN_OBJECT
+    )
+)
```

Comment:
This could break for future versions. You can compensate for that by using the annotation, which allows scanning the pattern fuzzy. Additionally, it might be beneficial to constrain the compatibility of this patch with targeting versions of Spotify Lite.